.github/workflows/commit-lint.yml

```diff
@@ -1,4 +1,4 @@
-name: "First commit message adheres to guidelines at https://goo.gl/p2fr5Q"
+name: First commit message adheres to guidelines at https://goo.gl/p2fr5Q
```

Comment:
do you want to remove the link since it's broken?

Reply:
I think that's outside the scope of this PR.